typing/ident.mli

```diff
@@ -74,6 +74,7 @@ val fold_name: (t -> 'a -> 'b -> 'b) -> 'a tbl -> 'b -> 'b
 val fold_all: (t -> 'a -> 'b -> 'b) -> 'a tbl -> 'b -> 'b
 val iter: (t -> 'a -> unit) -> 'a tbl -> unit
 val remove: t -> 'a tbl -> 'a tbl
+val compare_tbl: ('a -> 'a -> int) -> 'a tbl -> 'a tbl -> int
```

Comment:
Is this still needed?

Reply:
Not if we are happy with the latest iteration (without a custom comparison function), but I wasn't sure so I kept the work in history for now. I could still drop this commit. (It would probably be wise to replace `Ident.tbl` by `Map.Make(Ident)` anyway.)